lib/k6-summary/0.0.1/index.js

```diff
@@ -299,7 +299,11 @@ function summarizeMetrics(options, data, decorate) {
     }
   })
 
-  names.sort()
+  // sort all metrics but keep submetrics with their main metric
+  names.sort(function (a, b) {
+    // SPACE is the first non-control ascii character
+    return a.replace('{', ' ').localeCompare(b.replace('{', ' '))
```

Comment:
I also have a "cleaner" solution than string replace: extract the main und sub part of the metric, sort by main metric name and then sort by the sub metric name. The sort functions becomes a longer and more complex, but the approach would be "less hacky".